functional-test/src/test/groovy/org/rundeck/tests/functional/api/execution/ExecutionSpec.groovy

```diff
@@ -298,4 +302,145 @@ class ExecutionSpec extends BaseContainer {
+
+        def jobName2 = "xmljob2"
+        def jobXml2 = generateExecForEnabledXmlTest(jobName2)
+
+        def job1CreatedResponse = createJob(projectName, jobXml1)
+        assert job1CreatedResponse.successful
+
+        def job2CreatedResponse = createJob(projectName, jobXml2)
+        assert job2CreatedResponse.successful
+
+        when: "assert_job_execution_count with job 1"
+        CreateJobResponse job1CreatedParsedResponse = mapper.readValue(job1CreatedResponse.body().string(), CreateJobResponse.class)
+        def job1Id = job1CreatedParsedResponse.succeeded[0]?.id
+        def executions1 = doGet("/job/${job1Id}/executions")
+        JobExecutionsResponse parsedExecutionsResponseForExecution1 = mapper.readValue(executions1.body().string(), JobExecutionsResponse.class)
+        then:
+        parsedExecutionsResponseForExecution1.executions.size() == 0
+
+        when: "assert_job_execution_count with job 2"
+        CreateJobResponse job2CreatedParsedResponse = mapper.readValue(job2CreatedResponse.body().string(), CreateJobResponse.class)
+        def job2Id = job2CreatedParsedResponse.succeeded[0]?.id
+        def executions2 = doGet("/job/${job2Id}/executions")
+        JobExecutionsResponse parsedExecutionsResponseForExecution2 = mapper.readValue(executions2.body().string(), JobExecutionsResponse.class)
+        then:
+        parsedExecutionsResponseForExecution2.executions.size() == 0
+
+        when: "execute_job 1"
+        def jobExecResponseFor1 = executeJob job1Id
+        assert jobExecResponseFor1.successful
+        def executions1AfterExecResponse = doGet("/job/${job1Id}/executions")
+        JobExecutionsResponse parsedExecutionsResponseForExecution1AfterExec = mapper.readValue(executions1AfterExecResponse.body().string(), JobExecutionsResponse.class)
+        then: "assert_job_execution_count job1"
+        parsedExecutionsResponseForExecution1AfterExec.executions.size() == 1
+
+        when: "execute_job 2"
+        def jobExecResponseFor2 = executeJob job2Id
+        assert jobExecResponseFor2.successful
+        def executions2AfterExecResponse = doGet("/job/${job2Id}/executions")
+        JobExecutionsResponse parsedExecutionsResponseForExecution2AfterExec = mapper.readValue(executions2AfterExecResponse.body().string(), JobExecutionsResponse.class)
+        then: "assert_job_execution_count job1"
+        parsedExecutionsResponseForExecution2AfterExec.executions.size() == 1
+
+        when: "TEST: bulk job execution disable"
+        def idList = "idlist=${job1Id},${job2Id}"
+        def disabledJobsResponse = doPost("/jobs/execution/disable?idlist=${idList}", "{}")
+        assert disabledJobsResponse.successful
+
+        def executionsForJob1AfterDisable = doGet("/job/${job1Id}/executions")
+        JobExecutionsResponse parsedExecutionsResponseForExecution1AfterDisable = mapper.readValue(executionsForJob1AfterDisable.body().string(), JobExecutionsResponse.class)
```

Comment:
@DarwisNarvaezDev shouldn't the jobs be re-triggered here? before checking for number of executions? (with an expectation of failure)

Reply:
You're absolutely right! thanks.